### DISCUSSION TITLE
VAD5050 test: how to set location action coordinate conveniently

### DISCUSSION THREAD

**Discussion by vincentknw** [openTCS/opentcs#94] [Q&A]
when robot recieve a action of location, robot need to know the coordinate x y of location, but according to the doc i need to add a property of parameter.x ，paramater.y to location, is there any convenient way to add x y to action msg

**Comment by swltr** [comment]
@vincentknw Usually, I would expect that the vehicle would use the coordinates of the point that is linked to the location. Is that not the case here? Then yes, you would have to add properties for custom action parameters containing the coordinates.
